### ISSUE TITLE
Error_installing_nvimcom

### ISSUE THREAD

**Issue by DasHammett** [jalvesaq/Nvim-R#521]
Hi, 

I am getting the following error when trying to open R from vim (\rf):
```
Warning: invalid package ‘nvimcom_0.9-75.tar.gz’
Error: ERROR: no packages specified
```
My config:
R: 4.0.2
Linux Kernel 5.6.12
GCC 10.1.0

**Comment by jalvesaq** [comment]
The version of nvimcom in the last released Nvim-R is 0.9-92. So, you could try:

1. Completely uninstall Nvim-R.
2. Open R and do `remove.packages("nvimcom")`
3. Install Nvim-R again.

**Comment by DasHammett** [comment]
Thank you for the info.
Strangely enough, I closed vim and upon re-opening I did not have any issues.
Closing the issue
